instrumentation/netty/netty-4.1/build.gradle.kts

```diff
@@ -42,8 +42,8 @@ afterEvaluate{
 val versions: Map<String, String> by extra
 
 dependencies {
+    implementation("io.opentelemetry.instrumentation:opentelemetry-netty-4.1:${versions["opentelemetry_java_agent"]}")
     implementation("io.opentelemetry.javaagent.instrumentation:opentelemetry-javaagent-netty-4.1:${versions["opentelemetry_java_agent"]}")
```

Comment:
In other frameworks above, we have made this testImplementation.

Reply:
We can move it to `testImplementation` when our code in `src/main` does not reference OTEL code defined in the instrumentation module. Ideally, we would only have to depend on `io.opentelemetry.instrumentation:opentelemetry-netty-4.1` for compiling our main source set and use the  `io.opentelemetry.javaagent.instrumentation:opentelemetry-javaagent-netty-4.1` in our test runtime. However, we need compile access to things like the `NettyHttpServerTracer` in our main source set